packages/app/utils/api.ts

```diff
@@ -65,16 +66,24 @@ export async function respondAs(
     options: {
         format?: string
         httpStatusCode?: number // to pass a custom HTTP status code, defaults to `200`
+        payloadPath?: string | string[]
     }
 ) {
-    const { format = 'JSON', httpStatusCode = 200 } = options
+    const { httpStatusCode = 200 } = options
+    const responsePayload = options.payloadPath
```

Comment:
If we pass in a payload path, we'll use Lodash to get the value. Otherwise, use the default payload.